src/base/hide_password.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+
+/*
+ * hide_password.h -
+ */
+
+#ifndef	_PASSWORD_LOG_H_
```

Comment:
You changed entity name from password_log  to hide_password, then what about change the macro **PASSWORD_LOG_H** to _HIDE_PASSWORD_ as well.

Reply:
OK, I will fix it.
Thank you.